radicale/storage/multifilesystem.py

```diff
@@ -80,7 +82,7 @@ def remove(self, name):
     def text(self):
         components = (
             ical.Timezone, ical.Event, ical.Todo, ical.Journal, ical.Card)
-        items = set()
+        items = {}
```

Comment:
Oh, OK, the docstring of `_parse` was wrong about what was returned, I've fixed it.